cmd/manager/main.go

```diff
@@ -115,6 +123,20 @@ func main() {
 		log.Info(err.Error())
 	}
 
+	metricsServer := osdmetrics.NewBuilder().
+		WithPort(osdMetricsPort).
+		WithPath(osdMetricsPath).
+		WithCollectors([]prometheus.Collector{
+			localmetrics.RBACClusterwidePermissions,
```

Comment:
Why not use MetricsList here?

Reply:
@fahlmant The compilation error was

```
cmd/manager/main.go:128:17: cannot use localmetrics.RBACClusterwidePermissions (type *prometheus.GaugeVec) as type []prometheus.Collector in argument to "github.com/openshift/operator-custom-metrics/pkg/metrics".NewBuilder().WithPort(osdMetricsPort).WithPath(osdMetricsPath).WithCollectors
```

So I just matched the signature, to be honest.

Reply:
MetricsList from localnetrics is the right type, a slice of collectors.

Reply:
Oh, that's a good idea!